interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/SpelExpressionSuggester.scala

```diff
@@ -167,25 +170,36 @@ class SpelExpressionSuggester(expressionConfig: ExpressionDefinition[_], typeDef
 }
 
 private class NuSpelNodeParser(typer: Typer) extends LazyLogging {
-  private val parser = new org.springframework.expression.spel.standard.SpelExpressionParser()
-
-  def parse(input: String, variables: Map[String, TypingResult]): Try[NuSpelNode] = {
-    val rawSpelExpression = Try(parser.parseRaw(input))
-    rawSpelExpression.map { parsedSpelExpression =>
-      val collectedTypingResult = typer.doTypeExpression(parsedSpelExpression, ValidationContext(variables))._2
-      val nuAst = new NuSpelNode(parsedSpelExpression.getAST, None, collectedTypingResult)
-      nuAst
+  private val parser = new NuTemplateAwareExpressionParser(new SpelParserConfiguration)
+
+  def parse(input: String, language: String, position: Int, variables: Map[String, TypingResult]): Try[Option[(NuSpelNode, Int)]] = {
```

Comment:
It is not super important but since we have already 2 languages maybe `language` should not be `String` anymore? Maybe something that extends `sealed trait` so we know in patter matching that we are missing handlers for some languages?

Reply:
I feel you, but the Expression class that defines this values is used in over 100 places (as Intellij tells me), so it might bloat this MR... maybe next time? :D